util/test/sub_test

```diff
@@ -203,6 +203,11 @@ def LauncherTimeoutArgs(seconds):
 def ShellEscape(str):
     return re.sub(r'([\\!@#$%^&*()?\'"|<>[\]{} ])', r'\\\1', str)
 
+# Escape all special characters but leave spaces alone
+def ShellEscapeCommand(str):
+    return re.sub(r'([\\!@#$%^&*()?\'"|<>[\]{}])', r'\\\1', str)
```

Comment:
I copied the regexp from ShellEscape but removed the space. I'll change the argument name.